Coalition.rte/Devices/Explosives/FragGrenade/FragGrenade.ini

```diff
@@ -92,7 +92,7 @@ AddDevice = TDExplosive
 	ScriptPath = Coalition.rte/Devices/Explosives/FragGrenade/TimedFuze.lua
 	SpriteFile = ContentFile
 		FilePath = Coalition.rte/Devices/Explosives/FragGrenade/FragGrenade.png
```

Comment:
You deleted this file.
![uzira-confused](https://user-images.githubusercontent.com/50018358/127309084-eef90974-7a01-4c8c-abef-c0201487aa6f.png)

Reply:
Yeah, because I added extra frames? Did I not??